internal/mq/msgstream/mq_factory.go

```diff
@@ -86,12 +86,10 @@ type RmsFactory struct {
 	RmqBufSize     int64
 }
 
-// SetParams is used to set parameters for RmsFactory
-func (f *RmsFactory) SetParams(params map[string]interface{}) error {
-	err := mapstructure.Decode(params, f)
-	if err != nil {
-		return err
-	}
+// Init is used to set parameters for RmsFactory
+func (f *RmsFactory) Init(params *paramtable.ComponentParam) error {
+	f.RmqBufSize = 1024
```

Comment:
why rmqbufSize and ReceiveBufSize is not configurable?

Reply:
The old code is written as 1024 and this configuration does not exist in milvus.yaml

Reply:
> d code is written as 1024 and this configuration does not exist in milvus.yaml

Ok, let's leave it to next commit